arch/risc-v/src/bl808/hardware/bl808_timer.h

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ * arch/risc-v/src/bl808/hardware/bl808_timer.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_RISCV_SRC_BL808_HARDWARE_BL808_TIMER_H
+#define __ARCH_RISCV_SRC_BL808_HARDWARE_BL808_TIMER_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include "bl808_memorymap.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define BL808_TIMER_BASE(n) ((n == 0) ? BL808_TIMER0_BASE \
```

Comment:
Minor: wrap `n` with `()`.

Reply:
Well spotted. I have another change coming up that will edit this file, so I'll fix this as part of that.